### ISSUE TITLE
Lesson 12 Exercise 1 - "Angular Speed Has A Value Of 4" goals isn't checked right

### ISSUE THREAD

**Issue by clockworkprince** [GDQuest/learn-gdscript#579]
**Describe the bug**
When doing the first exercise in lesson 12, I get told that I don't have the variable `angular speed` set to 4 when I defined it inside of the function instead of outside.

**To Reproduce**
Steps to reproduce the bug:
1. Go to Lesson 12.
2. Click on the top exercise near the bottom of the lesson page.
3. Put in the following code: 
```
func _process(delta):
	var angular_speed = 4
	rotate(angular_speed * delta)
```
4. Click on the Run button to run the code.
5. The Checks section in the left column of the exercise screen will fail the "Angular Speed Has A Value Of 4" check.

**Expected behavior**
The Checks section should pass the "Angular Speed Has A Value Of 4" check instead. 

**Screenshots**
![GDscript From Zero - Bug Lesson 12 Exercise 1](https://user-images.githubusercontent.com/47702744/176881592-e12505b4-b2cf-46fa-957e-9f42f7ac524f.PNG)

**Information about your device (please complete the following information):**
 - Operating System: Windows 10 Home
 - Browser: Firefox 101.0.1

**Additional context**
None that comes to mind.

**Comment by NathanLovato** [comment]
As mentioned in the errors in the bottom-right, the variable isn't available script-wide. You need to define it outside the function. It's also explicitly mentioned in the practice goals.

![image](https://user-images.githubusercontent.com/12694995/176891595-e7940deb-ab80-4b50-a813-6d69ddcc2527.png)

**Comment by clockworkprince** [comment]
> 

That is correct, but I am not talking about checking the goal to put the variable in a script-wide position. I would have expected to have the top two goals passed, but the third failed. I understand why I failed this exercise, but I think the feedback I got for the manner in which I failed is confusing.

**Comment by clockworkprince** [comment]
> Did you struggle much to complete the practice after getting these two error messages?

No, but only because I realised that the top goal was marked wrongly as not being met. I have worked with GDscript in the past and already understand some of the fundamentals because of that. If that had not been the case, I likely would've been stuck and think that I had declared the `angular_speed` variable wrong somehow, while I had actually done it correctly.
